gradle.properties

```diff
@@ -23,4 +23,4 @@
 
 org.gradle.daemon=true
 org.gradle.parallel=true
-org.gradle.jvmargs=-XX:MaxPermSize=1024m
+org.gradle.jvmargs=-Xmx5120M
```

Comment:
> But I can also remove it here and create a separate ticket, since it has nothing to do with object grouping ;) What do you think?

Sounds great :+1: